modules/Bio/EnsEMBL/Compara/RunnableDB/BlastAndParsePAF.pm

```diff
@@ -122,7 +122,7 @@ sub fetch_input {
         #                           unannotated.fasta.01.pog
         my $fastafile = $self->param('blast_db');
 
-        foreach my $ext (qw(phr pin pog psd psi psq)) {
+        foreach my $ext (qw(phr pin pog psq)) {
```

Comment:
Does `makeblastdb` now generate only files with these extensions, or are there additional BLAST database files with suffixes not listed here?

Reply:
Let us handle this issue in a different mini-ticket.

Reply:
OK: ENSCOMPARASW-7616